lib/data/multipleBackendGateway.js

```diff
@@ -56,8 +56,8 @@ const multipleBackendGateway = {
                     dataStoreName: controllingLocationConstraint,
                     dataStoreType: client.clientType,
                 };
-                if (eTag) {
-                    dataRetrievalInfo.eTag = eTag;
+                if (backendParams) {
```

Comment:
if we just send back the versionId it would be clearer what is going on here and wouldn't have to do an object assign.